cities/laupheim.json

```diff
@@ -0,0 +1,199 @@
+                "title": "Kirchweihmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Oct 12 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88941,
+                    48.22771
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Kaltblutpferdemarkt:",
+                "location": "Festplatz Laupheim",
+                "opening_hours": "2017 Oct 12 08:00-16:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Weihnachtsmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": null,
+                "opening_hours_unclassified": "2017 Dez 14 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88757,
+                    48.23055
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Weihnachtsmarkt",
+                "location": "Schloss Großlaupheim",
+                "opening_hours": "2017 Nov 29-2017 Dec 01 16:00-22:00, 2017 Dec 02 12:00-22:00, 2017 Dec 03 12:00-20:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88941,
+                    48.22771
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Flohmarkt",
+                "location": "Festplatz Laupheim",
+                "opening_hours": "2017 Apr 08, 2017 May 13, 2017 Sep 09, 2017 Oct 07 08:00-16:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Flohmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Sep 10 08:00-16:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Laupheim",
+            "url": "https://www.laupheim.de/fileadmin/Dateien/Dateien/Freizeit_und_Kultur/Marktkalender_2017.pdf"
+        },
+        "map_initialization": {
+            "coordinates": [
+                9.883797,
+                48.229994
+            ],
+            "zoom_level": 12
```

Comment:
You can zoom in a bit more since the markets are quite near to each other, maybe `16` is a good value. You can launch the website locally using `python3 -m http.server`.